modules/clients/tests/GuestControllerTest.php

```
@@ -10,4 +10,332 @@
+    public function it_get_index_requires_guest_authentication(): void
+    {
+        // Arrange - No authenticated user
+        
+        // Act
+        // $response = $this->get('guest');
+        
+        // Assert
+        // $this->assertRedirect($response, 'sessions/login');
+        
+        $this->markTestIncomplete('HTTP test infrastructure needed');
+    }
```

PHPMD (2.15.0)

[error] The method it_get_index_requires_guest_authentication is not named in camelCase. (undefined)

(CamelCaseMethodName)